templates/takeovers/_ubuntu-masters-takeover.html

```diff
@@ -0,0 +1,15 @@
+{% with title="Ubuntu Masters: the&nbsp;masters&nbsp;speak",
+subtitle="Brendan Gregg, from Netflix, talks about Extended BPF and kernel-mode applications, the first change to how kernels are used in 50 years.",
+class="p-takeover--grad",
+header_image="https://assets.ubuntu.com/v1/7757c907-ubuntu-masters-logo-wht.svg",
+image_style="width: 270px; height: 150px;",
+image_hide_small=true,
+equal_cols=true,
+primary_url="https://www.youtube.com/watch?v=7pmXdG8-7WU",
+primary_cta="Watch the keynote",
+primary_cta_class="",
+secondary_url="/masters-conference/brendan-gregg?utm_source=takeover",
```

Comment:
I _think_ this needs to link to the `/masters-conference` landing page (which I'm currently working on).

Reply:
Yeah, I put that in as a placeholder but I'll update it and change this PR to a draft